File: totosuki/ABC/B/121.py
N, M, C = map(int, input().split())
B = list(map(int, input().split()))
cnt = 0

for _ in range(N):
  tmp = C
  A = list(map(int, input().split()))
  
  for i in range(M):
    tmp += A[i] * B[i]

  if tmp > 0:
    cnt += 1

print(cnt)

File: OCH_Tao/ABC/A/121.js
const input = (require("fs").readFileSync("/dev/stdin", "utf8").trim().split("\n"));
const [H, W] = (input[0].split(" ")).map(x => parseInt(x, 10));
const [h, w] = (input[1].split(" ")).map(x => parseInt(x, 10));
console.log((H - h) * (W - w));